data/json/items/generic.json

```
@@ -2832,6 +2832,20 @@
     "volume": "850 ml",
     "flags": [ "SLEEP_AID" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "shark_plushie",
+    "symbol": "s",
+    "color": "cyan",
+    "name": { "str": "shark plushie" },
+    "description": "A somewhat large, mass produced shark plushie.  Meant to provide comfort to people of all ages.",
+    "price": 600,
+    "price_postapoc": 20,
+    "material": [ "cotton" ],
+    "weight": "640 g ",
+    "volume": "1300 ml",
```

```suggestion
    "volume": "30000 ml",
```
Not super scientific, but if IKEA says it's 1.03m long, and if you treat it like a cylinder about 0.2m wide, the volume would be about 30 liters